src/components/carousel/shared-components/CarouselCards.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 
 const CarouselCards = ({ images, current, paginationDots }) => {
-  const carouselCards = images.map((image, index) => (
-    <div key={index} className={index === current ? 'carousel_card carousel_card-active' : 'carousel_card'}>
-      <img className='card_image' src={image.image} alt='' />
+  const carouselCards = images.map((carousel, index) => (
```

Comment:
Just make sure to auto format all docs before committing

Reply:
cleaned code